File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        String check1 = String.valueOf(a) + String.valueOf(b);
        String check2 = String.valueOf(b) + String.valueOf(a);
        int num1 = Integer.parseInt(check1);
        int num2 = Integer.parseInt(check2);
        int answer = num1 > num2 ? num1 : num2;
        return answer;
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    check1 = str(a) + str(b)
    check2 = str(b) + str(a)
    num1 = int(check1)
    num2 = int(check2)
    if check1 > check2:
        answer = check1
    else:
        answer = check2
    return int(answer)